config.org

```diff
@@ -145,9 +146,15 @@ This is an adjustment to the default hook, which hides drawers by default
       '((:results . "replace code")
         (:wrap . "SRC example")))
 #+END_SRC
-
+** OSC52 (copy over ssh/terminal)
+#+BEGIN_SRC elisp
+(after! osc52e
+  (osc52-set-cut-function)
```

Comment:
`osc52-set-cut-function` needs to run during init so it knows how to `yank-to-clipboard` over ssh/osc52.